packages/palette/tsconfig.ios.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "./*": ["./*.ios", "./*"],
+      "../*": ["../*.ios", "../*"]
```

Comment:
What's this one for?

Reply:
I meannn... `./` and `../` are different paths technically.

Reply:
Oh, just wondering  why  it's needed if the `baseUrl` points to the root?